test_helpers/test_helpers.go

```diff
@@ -438,21 +442,36 @@ func WaitForVirtualGuestToHaveNoActiveTransactions(virtualGuestId int) {
 
 func WaitForVirtualGuestToHaveNoActiveTransactionsOrToErr(virtualGuestId int) {
 	virtualGuestService, err := CreateVirtualGuestService()
-    if err != nil {
-        return
-    }
+	if err != nil {
+		return
+	}
 
 	fmt.Printf("----> waiting for virtual guest to have no active transactions pending\n")
 	Eventually(func() int {
 		activeTransactions, err := virtualGuestService.GetActiveTransactions(virtualGuestId)
-        if err != nil {
-            return 0
-        }
+		if err != nil {
+			return 0
+		}
 		fmt.Printf("----> virtual guest: %d, has %d active transactions\n", virtualGuestId, len(activeTransactions))
 		return len(activeTransactions)
 	}, TIMEOUT, POLLING_INTERVAL).Should(Equal(0), "failed waiting for virtual guest to have no active transactions")
 }
 
+func SshKeyPresent(sshKeyId int) bool {
+	accountService, err := CreateAccountService()
+	Expect(err).ToNot(HaveOccurred())
+	sshKeys, err := accountService.GetSshKeys()
+	Expect(err).ToNot(HaveOccurred())
+
+	present := false
+	for _, sshKey := range sshKeys {
+		if sshKey.Id == sshKeyId {
+			present = true
+		}
```

Comment:
why don't you break here? what's the point of continuing the search once you found the key?